llama-index-integrations/postprocessor/llama-index-postprocessor-jinaai-rerank/pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = ["poetry-core"]
+
+[tool.codespell]
+check-filenames = true
+check-hidden = true
+skip = "*.csv,*.html,*.json,*.jsonl,*.pdf,*.txt,*.ipynb"
+
+[tool.llamahub]
+contains_example = false
+import_path = "llama_index.postprocessor.jinaai_rerank"
+
+[tool.llamahub.class_authors]
+JinaRerank = "llama-index"
```

Comment:
feel free to add your github username here if you like!

Reply:
done, thanks